indico/modules/events/registration/controllers/display.py

```diff
@@ -325,7 +325,7 @@ class RHRegistrationDisplayEdit(RegistrationEditMixin, RHRegistrationFormRegistr
 
     def _check_access(self):
         RHRegistrationFormRegistrationBase._check_access(self)
-        if not self.registration.can_be_modified:
+        if self.registration and not self.registration.can_be_modified:
```

Comment:
I don't see how this code ever gets called - `_process_args` will redirect away if there is no registration. So I have the feeling your patches are calling `_check_access` for some reason even though `_process_args` already returned a response which should stop further processing...

Reply:
ok will investigate, I think it's because of PMS